src/frontend/src/index.ts

```diff
@@ -75,9 +74,13 @@ const init = async () => {
   // https://github.com/dfinity/internet-identity#build-features
   showWarningIfNecessary();
 
-  // Custom routing to the FAQ page
+  // Redirect to the FAQ
+  // The canister should already be handling this, this is just a safety
+  // measure
   if (window.location.pathname === "/faq") {
-    return faqView();
+    const faqUrl =
+      "https://support.dfinity.org/hc/en-us/sections/8730568843412-Internet-Identity";
```

Comment:
I did originally, but then it turned out to be a an extra file for nothing (only 2 uses; the 3 one in the frontend to an actual FAQ entry has a different URL). I've added to my todolist to actually add the constant to e.g. the .did file or something so that at least the canister can benefit from it as well.